tests/test_authority.py

```diff
@@ -60,7 +60,7 @@ class TestAuthority(unittest.TestCase):
     # discovery.
     nonHardCodedAuthority = 'https://login.doesntexist.com/' + cp['tenant']
     nonHardCodedAuthorizeEndpoint = nonHardCodedAuthority + '/oauth2/authorize'
-    dstsTestEndpoint = 'https://test-dsts.core.azure-test.net/dstsv2/common'
+    dstsTestEndpoint = 'https://test-dsts.dsts.core.azure-test.net/dstsv2/common'
```

Comment:
Hi Jared, thanks for addressing the earlier review feedback.

On a close look, it seems even the original `authority.py` implementation BEFORE this PR would still work for your this newly adjusted test cases. No? So why would we need to adjust them to a potentially oversensitive ".dsts." pattern match?

Reply:
These test cases were wrong before. Every dsts instance will fit the pattern *-dsts.dsts.core.* and test-dsts.core.azure-test.net doesn't meet this requirement so the test cases were failing with my new implementation. The new authority.py implementation for dSTS is to support new cloud environments in which we will not be allowed to explicitly whitelist endpoints in public GitHub repos due to customer restrictions on releasing DNS names. Since I have removed dSTS whitelisting constants, these test cases should be sufficient to cover all current and future dSTS endpoints that fit the *.dsts.* DNS address pattern.

Reply:
Thanks for the explanation! This PR is good to merge then.

Do you have anything else to be added, either in this PR or in a subsequent PR? Whether/when do you want a new ADAL Python release with these change?

Reply:
I don't have any other changes to add. This should cover our current needs. Do you have a release cadence? If you have a release coming up we can wait for that, if not it would be good to get a release out so we have it available by December.